lectures/debugging-R.md

```diff
@@ -5,6 +5,8 @@ title: Debugging
 
 ## Setup
 
+* **Send `debugging-example-unfixed.R` to students so
+  they`debugging-example-unfixed.R`**
```

Comment:
Not sure what the second part/line, "so / they`debugging-example-unfixed.R`", adds.